This patch transforms the BSD-style Makefile into a 
something appropriate for a Debian Linux.
--- a/Makefile	2020-07-26 14:37:17.440815279 +0200
+++ b/Makefile	2020-07-26 14:39:24.799507841 +0200
@@ -23,10 +23,10 @@
 #OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE 
 #OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-PREFIX?=/usr/local
-INSTALLBIN=$(PREFIX)/bin
-INSTALLSHARE=$(PREFIX)/share/games
-INSTALLMAN=$(PREFIX)/share/man
+DESTDIR?=/usr/local
+INSTALLBIN=$(DESTDIR)/bin
+INSTALLSHARE=$(DESTDIR)/share/games
+INSTALLMAN=$(DESTDIR)/share/man
 
 CC?=gcc
 AR?=ar
@@ -72,7 +72,6 @@
 OBJ_TOPLEVEL=${SOURCES_TOPLEVEL:.c=.o}
 
 all:	dMagnetic
-	strip dMagnetic
 
 clean:
 	rm -rf dMagnetic
@@ -91,8 +90,8 @@
 	cp dMagnetic $(INSTALLBIN)
 	cp dMagnetic.1 $(INSTALLMAN)/man1
 	cp dMagneticini.5 $(INSTALLMAN)/man5
-	cp README.txt $(INSTALLSHARE)/dMagnetic
-	cp LICENSE.txt $(INSTALLSHARE)/dMagnetic
+	#cp README.txt $(INSTALLSHARE)/dMagnetic
+	#cp LICENSE.txt $(INSTALLSHARE)/dMagnetic
 	cp dMagnetic.ini $(INSTALLSHARE)/dMagnetic
 
 
@@ -100,7 +99,7 @@
 	$(LINK) $(LDFLAGS) -o $@ $(OBJ_LOADER) $(OBJ_LINEA) $(OBJ_VM68K) $(OBJ_GUI) $(OBJ_TOPLEVEL)
 
 .c.o:
-	$(CC) $(CFLAGS) $(CFLAGS_EXTRA) $(INCFLAGS) -c -o $@ $<
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(CFLAGS_EXTRA) $(INCFLAGS) -c -o $@ $<
 
 
 ## in case some post-compilation checks are needed
